### PR TITLE
Add slidebar and refactor index.html

### PR DESCRIPTION
**Pull request by starcalibre**
Getting close to adding the filter functionality now!

* Add the slidebars plugin, and rewrite structure of the index.html and various other jQuery methods so everything still functions well with the slidebar.
* Remove the tabbed navigation, and move the view control to the navbar. This frees up screen real estate and makes everything look a bit cleaner.
* Fix a small bug in the cluster scatterplot that causing an empty SVG to be added to the DOM when the number of clusters was changed.

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -119,6 +119,7 @@
     background: rgba(0, 0, 0, 0.8);
     color: white;
     border-radius: 2px;
+    z-index: 999999;
```
Comment (jni) [inline public/css/style.css]: Is this assuming that we won't have > 1M points?

**Comment by jni** [comment]
Hmm, tried to play with this... `node server.js`... Navigate to localhost:8080... I get a page greeting me with a friendly "Yo", as well as an empty drawer for "Select screen"... Suggestions?

**Comment by starcalibre** [comment]
Run ``bower install && npm install`` to make sure you've got all the
dependencies. If it still doesn't work, can you check for any errors in
your browsers console?

File: public/css/style.css (hunk 1 above)
Comment (starcalibre) [inline public/css/style.css]: It's just an arbitrarily large number. Though I assume we won't be drawing
1M SVGs! Datasets that large would be pointless viewed on a scatterplot
anyway. That's something where 2D binning would be useful :)

**Comment by jni** [comment]
That is of course what was wrong! Looks *great!* And I need to get in the habit of checking the console, that would probably have told me. imho browsers should have a little "hey all sorts of shit is happening in the JS console" button that lights up when things go wrong. =)

**Pull request merged**